source/digits_hits/include/GateCrystalHit.hh

```diff
@@ -85,6 +86,10 @@ private:
   G4double m_scannerRotAngle; // Rotation angle of the scanner
   GateOutputVolumeID m_outputVolumeID;
   G4int m_systemID;           // system ID in for the multi-system approach
+  G4double m_energyInitial;
```

Comment:
Could you add a one line comment to those new lines?
What is the difference between initial and generated energy?